kingdoms/map.cpp

```diff
@@ -1062,17 +1062,13 @@ void ARegionList::SetFractalTerrain(ARegionArray *pArr)
 
 void ARegionList::NameRegions(ARegionArray *pArr)
 {
-	int seed = 0;
-	int grown = 0;
```

Comment:
All of the variables in the function are set and incremented but never actually used, so the MacOS X compiler (CLANG) complained about them since it's static analysis code is stronger than the standard GCC code.